debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbStreamingChangeEventSource.java

```diff
@@ -77,22 +73,21 @@ public MongoDbStreamingChangeEventSource(MongoDbConnectorConfig connectorConfig,
 
     @Override
     public void init(MongoDbOffsetContext offsetContext) {
-
         this.effectiveOffset = offsetContext == null ? emptyOffsets(connectorConfig) : offsetContext;
     }
 
     @Override
-    public void execute(ChangeEventSourceContext context, MongoDbPartition partition, MongoDbOffsetContext offsetContext)
+    public void execute(ChangeEventSourceContext context, MongoDbPartition partition, MongoDbOffsetContext unused)
```

Comment:
@jpechane  this was previously changed by @mfvitale  to use the `effectiveOffset` field set in the `init` method. I'm just making it cleaner that this parameter is not used.

Reply:
@jcechace Instead of renaming the var it might be better to add a JavaDoc and explain why it is not used

Reply:
The renaming in some IDEs helps by hinting that this is unused on purpose. However I can surely add the docs. 

BTW: I think this is a case for majority of our connectors currently.